docs/documenting/device-info.md

```diff
@@ -0,0 +1,36 @@
+# Twomes Device Info page
+
+The purpose of a device info page is allow subjects and other people that encounter your measurement device is to find more information what the device is doing easily. The contents of the folder [/docs/deviceinfo/](/docs/deviceinfo/) of a Twomes measurement device repository can be used as the template for this. On a [M5Stack CoreInk](https://github.com/m5stack/M5-CoreInk) device, after the device was provisioned, the e-ink screen can show a QR-code that leads users directy to the device info page.  
+
+## Prerequisites
+You only need a web browser and a plain text editor (or your favourite IDE).
+
+## Writing
+Adapt the content of `index.html` to fit the needs of your measurement device type. Be sure to write in the language the intended audience of the subjects and other pople that might encounter your measurement device. The 
+.[Twomes provisioning app](https://github.com/energietransitie/twomes-app-warmtewachter). After scanning its QR-code, the manual of your device type is presented to the user by the [Twomes provisioning app](https://github.com/energietransitie/twomes-app-warmtewachter). 
```

Comment:
@henriterhofte This paragraph has some incorrect sentences, but I am not sure as to what they should be.

Reply:
fixed n https://github.com/energietransitie/twomes-generic-esp-firmware/pull/61/commits/9f0fba3142314d61b2eb14cf0c75a22ecd945cb5